pages/instructorGradebook/instructorGradebook.ejs

```diff
@@ -95,7 +95,16 @@
         </div>
 
         <div class="card-footer">
-          Download <a href="<%= urlPrefix %>/gradebook/<%= csvFilename %>"><%= csvFilename %></a>
+          <p>Download <a href="<%= urlPrefix %>/gradebook/<%= csvFilename %>"><%= csvFilename %></a></p>
+          <small>
+            <strong>Roles:</strong>
```

Comment:
This text is awesome! Can we also add this to the bottom this table: https://github.com/PrairieLearn/PrairieLearn/blob/master/pages/instructorAssessmentInstances/instructorAssessmentInstances.ejs#L100

This table also has a `Role` column.

Reply:
Done in https://github.com/PrairieLearn/PrairieLearn/pull/2008/commits/e44bbadb57e1af37fdecd842a219e24bdf6cc0fc